### PR TITLE
DTSPO-17094: Artifactory Patching Doc

### PR DESCRIPTION
**Pull request by DJ-Khaled**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/DTSPO-17094

### Change description ###
Artifactory Patching Documentation 

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by ConnorOKane-Kainos — APPROVED** [review]
LGTM :)

File: source/aks/patching-artifactory.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+1. Azure CLI Login and Kubernetes Services:
+
+• Access Azure CLI and search for Kubernetes Services
+
+• Type the following PTL in the search box:
+
+![](Images/artifactory_pic01.png)
+
+2. Connect to PTL Box:
```
Comment (endakelly) [inline source/aks/patching-artifactory.html.md.erb:32]: PTLSBOX

File: source/aks/patching-artifactory.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+patchesStrategicMerge:-../../artifactory/artifactory-sbox.yaml
+```
+
+7. Push Changes:
+
+• Raise a PR and push changes out for review and approval
+
+8. Update Version:
+
+• For the PTL box, remove "artifactory-sbox.yaml" file and its corresponding code from kustomization.yaml
```
Comment (endakelly) [inline source/aks/patching-artifactory.html.md.erb:113]: PTLSBOX

File: source/aks/patching-artifactory.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+9. Push Changes:
+
+• Raise a PR and push changes out for review and approval
+
+10. Troubleshooting:
+
+• If errors occur, troubleshoot as follows:
+	• Add the missing namespace "ARTIFACTORY" in the file "admission-control.yaml" in the downloaded repo "chart-neuvector"
+	• Raise a PR and push changes out for review and approval
+
```
Comment (endakelly) [inline source/aks/patching-artifactory.html.md.erb:135]: This was a one time fix, noted in https://hmcts.github.io/ops-runbooks/aks/updating-neuvector-policies.html. Best to just link out to that

File: source/aks/patching-artifactory.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+• Raise a PR and push changes out for review and approval
+
+Troubleshooting Steps:
+
+• To resolve namespace issues in "chart-neuvector":
+	• Delete specific files from the azure file share
+	• Files to delete: "admission_control.backup" and "crd.backup"
+	• Delete the HR on the cluster and let it redeploy
+	• Confirm proper functioning after redeployment
+
```
Comment (endakelly) [inline source/aks/patching-artifactory.html.md.erb:153]: Same as https://github.com/hmcts/ops-runbooks/pull/224/files#r1523293794

File: source/aks/patching-artifactory.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+          name: artifactory-oss
+          env:
+```
+
+[Example PR] (https://github.com/hmcts/cnp-flux-config/pull/29665/files)
+
+9. Push Changes:
+
+• Raise a PR and push changes out for review and approval
+
```
Comment (endakelly) [inline source/aks/patching-artifactory.html.md.erb:129]: Can you add something around going to the artifactory site after the upgrade and checking the packages and repositories are still there to make sure it's working? https://artifactory.sandbox.platform.hmcts.net/ui/repos/tree/General/

**Pull request merged**